_posts/en/newsletters/2021-07-21-newsletter.md

```diff
@@ -0,0 +1,154 @@
+---
+title: 'Bitcoin Optech Newsletter #158'
+permalink: /en/newsletters/2021/07/21/
+name: 2021-07-21-newsletter
+slug: 2021-07-21-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes recent changes to services and client
+software, discusses why wallets should wait before generating taproot
+addresses, lists new software releases and release candidates, and
+summarizes notable changes to popular Bitcoin infrastructure software.
+
+## News
+
+*No significant news this week.*
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Lightning-powered news site Stacker News launches:**
+  [Open source][stacker news github] news site [Stacker News][stacker.news]
+  launched allowing LNURL authentication as well as voting and commenting using
+  LN micropayments.
+
+- **Suredbits announces DLC wallet alpha release:**
+  [Suredbits' bitcoin-s][suredbits blog] software includes a GUI and allows the
+  execution of Discreet Log Contracts (DLCs) on the Bitcoin blockchain using an
+  oracle. The announcement concludes by mentioning they also plan to use [schnorr
+  signatures][topic schnorr signatures] and [Point Time Locked Contracts
+  (PTLCs)][topic ptlc] to implement [DLCs on Lightning][suredbits blog dlcs ln].
+
+- **Sparrow 1.4.3 supports P2TR:**
+  Sparrow's [1.4.3 release][sparrow 1.4.3] supports [single-sig P2TR
+  wallets][taproot series 4] on [signet][topic signet] and regtest. The release
+  also supports [sending to P2TR addresses][taproot series 1].
```

Comment:
Should P2TR addresses be swapped to bech32m per the style guide change above?

Reply:
From a quick look at their [diff](https://github.com/sparrowwallet/drongo/commit/967a2c2026f15a6b458d055e33ac488e53df65fd#diff-85d66c4a8decae8670cf725b591dfa8c626a569ed607aa8e887964795a9df9b8), it seems to me that they only added support for bech32m with v1 segwit (P2TR).  So what you wrote is not only correct but precise.  That said, it would be more in line with the style guide suggestion to say something like "sending to bech32m addresses for P2TR".  Your choice.

Reply:
Took this suggestion